### PR TITLE
Add enable minor version upgrades flag to upgrade policy

### PR DESCRIPTION
**Pull request by tzvatot**
@zgalor PTAL

### CODE REVIEW

File: model/clusters_mgmt/v1/upgrade_policy_type.model [hunk 1]
```diff
@@ -33,4 +33,7 @@ class UpgradePolicy {
 
 	// Next time the upgrade should run.
 	NextRun Date
+
+	// Indicates if minor version upgrades is allowed for automatic upgrades (for manual it's always allowed).
+  EnableMinorVersionUpgrades
```
Comment (zgalor) [inline model/clusters_mgmt/v1/upgrade_policy_type.model:38]: indentation issue

File: model/clusters_mgmt/v1/upgrade_policy_type.model [hunk 2]
```diff
@@ -33,4 +33,7 @@ class UpgradePolicy {
 
 	// Next time the upgrade should run.
 	NextRun Date
+
+	// Indicates if minor version upgrades is allowed for automatic upgrades (for manual it's always allowed).
```
[the remark this answers is not in the document]
Comment (tzvatot) [reply model/clusters_mgmt/v1/upgrade_policy_type.model]: Fixed

File: model/clusters_mgmt/v1/upgrade_policy_type.model (hunk 1 above)
Comment (tzvatot) [reply model/clusters_mgmt/v1/upgrade_policy_type.model]: Fixed

**Pull request merged**